buildSrc/src/main/groovy/openhouse.maven-publish.gradle

```diff
@@ -49,6 +54,16 @@ publishing {
         artifact javadocJar
         artifact jar
 
+        configurePom(pom)
+      }
+    } else if(tasks.findByName('bootJar')) {
+      mavenJava(MavenPublication) {
```

Comment:
can we add a one line comment saying this block is only for service modules?

Reply:
let me think about in a separate PR, this file is missing it for other conditions